### PR TITLE
Final fixes

### PR DESCRIPTION
**Pull request by bhubhik**

### CODE REVIEW

File: blogs.njk [hunk 1]
```diff
@@ -76,11 +76,11 @@ renderData:
       </ul>
     </div>
   </div>
-  <div class="col-span-12 md:col-span-4 mx-auto w-full md:h-screen md:relative mt-4 flex flex-col items-center md:items-start md:-ml-16 ">
+  <div class="col-span-12 md:col-span-4 mx-auto w-full md:h-screen md:relative mt-4 flex flex-col items-center md:items-start md:-ml-16 md:top-64 ">
```
Comment (geshan) [inline blogs.njk:79]:
```suggestion
  <div class="col-span-12 md:col-span-4 mx-auto w-full md:h-screen md:relative mt-4 flex flex-col items-center md:items-start md:-ml-16 md:top-32">
```

32 should be ok.

Comment (geshan) [reply blogs.njk:79]: that is `8rem` I believe.

Comment (bhubhik) [reply blogs.njk]: Yes it is. Will be changed on next push

File: _includes/footer/footer.njk [hunk 2]
```diff
@@ -6,7 +6,7 @@
             <ul class="flex">
                 {%- for entry in collections.all | eleventyNavigation %}
                 <li class=" text-white hover:text-gray-300 text-lg pr-4 cursor-pointer font-semibold ">
-                    <a href="#">{{ entry.title }}</a>
```
Comment (geshan) [inline _includes/footer/footer.njk:9]: :)

**Pull request merged**